pkg/api/v1/blueprint_types.go

```diff
@@ -0,0 +1,69 @@
+package v1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
🚫 **[checkstyle]** <goimports> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>File is not `goimports`-ed